Extensions/tweaks.js

```diff
@@ -152,6 +152,11 @@ XKit.extensions.tweaks = new Object({
 			default: false,
 			value: false
 		},
+		"wide_sources": {
+			text: "Increase max width of post sources to avoid truncation",
+			default: false,
```

Comment:
Made it false out of abundance of caution, but I do think it's an "everyone should be using this" kind of thing, so I'll swap